### DISCUSSION TITLE
Moderation: Sharing the load, Sharing tickets, feedback and more.

### DISCUSSION THREAD

**Discussion by thefuturebird** [mastodon/mastodon#22979] [Q&A]
I have two suggestions. One would be easy to implement, the other much more complex, but they both address the same issue: sharing the work of moderation and making moderation a smoother less stressful and dramatic process.

Ideas:
1. server-whitelisted admin/mod discussion forums
2. shared post report ticket system

**server-whitelisted admin/mod discussion forums**
Right now many mods and admins use the hashtag #fediblock to share information about bad actors, bad servers, DDOS attacks, circumventing blocks, racism, sexism and transphobia. It's a smart adhoc tool that just barely works and it also causes a lot of drama.

Mods and admins have discussed and set up spaces to talk about these things in discord, and on other services, but it's also adhoc, exclusionary and hard to get in to the loop.

Could Mastodon generate credentials that identify which users are mods/admins? Could these be used to enter another designated service (or the DMs system in Mastodon, although I don't think the DM system is secure enough) along with a server white-list?

There would not be just one big discussion space but several. You whitelist the mods/admins who are enforcing similar rules and who have similar values-- Mods and admins could broadcast "fediblock" type information to their whitelist. Discuss best actions etc.

**shared post report ticket system**
Simply getting the right people connected for these discussions is the first and most critical part, it would be ideal if this kind of communication and sharing were built in to the moderation system itself. Again, admins and/or mods would make a white list of servers. Decide if white-listing needed to be mutual for sharing to occur or not. When a user makes a report it would go first to their local instance, the mods/admins would have the option of sharing that report to their white list. 

Each sever would decide if they want to import all blocks, suspensions, and limiting from each server, or review and approve reports as they come in. At first this would make a little more work... but then as more trust was built it would make much less work. 

Comments/notes on reports would be visible to every server that sharers the report. 

Lastly the mods/admins at the sever where the report was made would have the *option* of notifying the user who made the report about what happened. This kind of feedback is essential to helping users to feel like their reports don't just disappear in to a black hole. Complex moderation questions might not merit such a response, but if I get spammed with the n-word and report it-- then mute the account I don't know if my report mattered or not. A simple "yes that WAS bad, it was good that you reported it and we acted" builds trust and community.

**Comment by doctormo** [comment]
(This is the first time I've posted to the mastodon gitlab, my name is Martin, I'm an Inkscape developer and my qualifications are that I wrote the moderation tools for the inkscape website and forums in concert with experienced volunteer moderators)

The design of any moderation cooperation tool is going to need quite a bit of careful designing. And probably a lot of testing to see what it's limits and network effects are.

One thing I see as missing in moderation is that a user can report a post, causing an admin or moderator work, but they can not add concern to a post which will trigger a report if someone else(s) also raise some concern. Aggregating decisions from the users is a solid way of lowering the bar to report and increasing the bar to end up as a report.

I bring up this idea from the left field because it's similar to the idea of aggregating decisions, concerns, bans and so on between moderators and instances (and I think of these levels differently) might produce a design that allows mastodon moderators to cooperate without communicating black and white ban or not ban, etc.

I would tackle this problem by creating an online design unconference and invite existing and experienced mastodon moderators, some good ux and ui designers and a couple of meeting organisers (if you have the funds). You could progress the field of online moderation through this method.

**Reply by nottrobin** [reply]
I've also long felt that content moderation should be a crowd-sourced endeavour, although it would of course take careful design. Is Wikipedia perhaps a good model for this? AFAIK all discussions on Wikipedia content happen in the open, while mods are still the decision makers.

I'd love to see notices above toots saying "this toot has been flagged X times as misinformation" or something. I'd also love to be able to see the discussion history behind moderation decisions.

Do you think maybe this topic should have its own thread? The idea of aggregating user opinion doesn't feel directly related to @thefuturebird's original point. It could be complementary to the idea of federated moderation, as it could be to single-instance moderation.

**Reply by thefuturebird** [reply]
I think having a "concern" option for users fits with this topic since it would work better if "concern" could be shared using the white list system I described above. So the concerns of users from multiple instances could accumulate and bring the attention of mods/admins.

So, it's a "shared ticket" issue.

On another topic "crowd sourced" moderation isn't an idea I'm a fan of. It's too easy of a vector for abuse, and tends to magnify prejudices, and imbalances rather than, well, moderating them. It also can feel like trying to pass off the work of keeping a community safe on to the shoulders of the victims of the abuse. Maybe there are better ways to design it, and if you start a thread and outline how it might work I'd be happy to hop into that conversation. Just be aware that "let's crowd source it" has kind of a bad history and reputation as ... a kind of minimization of the issue in some recent conversations (though I don't think that is what you are doing here nottrobin) it's kind of like when people say "well just block them" without recognizing that having white supremacists, or people who don't think trans people should exist is a bigger problem that needs to be addressed at the community level (at the level of the instance)-- I hope that makes sense!

**Reply by nottrobin** [reply]
> I think having a "concern" option for users fits with this topic since it would work better if "concern" could be shared using the white list system I described above. So the concerns of users from multiple instances could accumulate and bring the attention of mods/admins.

I see what you mean, yes this could very well work together.

> Just be aware that "let's crowd source it" has kind of a bad history and reputation as ... a kind of minimization of the issue in some recent conversations

Yes point taken. I am kind of aware of this, and I completely agree that pure and direct crowd-sourcing won't work at all. Maybe I'm overoptimistic that this can be overcome. I was hoping that if you proceed in full knowledge of this risk then you can include crowd-sourcing with the correct checks and balances. Otherwise don't you end up with the same problems that always come when you disempower communities of people? Mod making decisions that completely ignore a significant body of opinion just because it's easier for them (as opposed to them deliberately rejecting a significant body of opinion because it's racist or divisive or ... any other actual reason, which is totally legit).

Basically, I think "crowdsource it" in the cases you're talking about is a push for simplicity ("why don't you *just* crowdsource it"), whereas what I'm considering would certainly be a system of considerable complexity.

Do you not think Wikipedia is a fairly got example of this done quite well? There's a huge number of people and a lot of incentive for people to deface and corrupt content on Wikipedia, but the quality remains high. I think it's a hugely complex moderation system, and so might it have to be with Mastodon, but I think it works doesn't it? Or do you think crowdsourcing gets used as a scapegoat to justify abuse / minimise voices there too?

**Reply by thorhop** [reply]
Maybe there's a middleground here. Off the top of my head I can think of how certain communities run by nefarious actors could mobilize their fanbase to report someone out of existence. Social engineering can happen on a community scale, and in that case, perhaps certain servers, admins and mods should in turn be reported - for abusing their influence with malicious intent and being penalized in some way to discourage abuse.

Who reports the reporters?

I think there should also be public standards regarding specific parameters around reporting and complaints so that reports can be tangentaly quantified and percieved with no confusion or secrecy. Why one got muted/banned, why someone got reported, how long the effects have lasted and will last, the content or post in question cached for posterity (maybe through a BASE64 screenshot or something) to properly facilitate complaints out in the open and transparently. Perhaps the admin of a server has malicious intent and tries to wipe the evidence of a false complaint. 

Perhaps something like a discussion about standard timeouts should be held, like how long you get muted/banned for infractures, what compounding effects are there when infractures continue to happen, as well as timeout levels according to severity, etc. For instance 1 case of inciting to violence would lead to say 2 weeks as an example, but then after the ban the person does it again and gets 4 weeks - based upon the fact that the infracture is the same.

There shouldn't be any confusion is what I'm saying. The key here is transparency, unlike the moderation on Insta, YouTube, etc which have more emotionally based moderation depending upon how the company feels that day. 
Abuse needs to be prevented in all directions and I think full transparency is the right way to go. The users have to be informed, the mods have to be informed, the admins need to be informed, everybody needs to be able to see what if any disputes there are regarding someone being reported. 

But then again maybe it should be possible to request private reviews in certain cases, for some legit reason of privacy should be observed. How do you then make sure that the case isn't being tried by someone who favours the one filing the complaint? Do we create a witness seeding program, where moderators are chosen at random to review cases using identifying things about moderators like location, vocation and association to exclude possible contaminants to that pool? 

In the end of it all I think it's not about "crowd funding", but democratizing the complaint process with a focus to prevent abuse and corruption across the board. That is a _huge_ nut to crack, which human societies have tried to do through the judicial systems. I think we therefore have to think about the problem of protecting peoples rights, as well as preventing abuse of ToS, CoC or ethical clauses, because we need some way to identify, quantify and justify the effects of penalization. 

It has to be just.

**Comment by trwnh** [comment]
something like combining #15526 with relays would probably work for the 2nd point. for the 1st, i assume the introduction of groups might help.

**Comment by ethergear** [comment]


**Reply by nottrobin** [reply]
I like a lot of what you're saying, but I'd personally vote strongly against intentionally basing any part of Mastodon infrastructure on a service from one of the big tech monopolies, regardless of how useful its features might be. That sounds very obviously in conflict with the goals of Mastodon and the fediverse to me.

**Comment by supernovae** [comment]
Most of my mod activity is federated instances, not necessarily local stuff.  I delete the random MAGA account once a week, but otherwise I end up dealing with a lot of adult content more than anything else.  

Since the moderation of the fediverse is so one sided, I think one thing we should talk about is what it means to moderate a one-sided network. By one sided, it means Mods only see one side of the story when posts are reported to moderators. If they're local, we can discover both sides - but this is exceedingly rare.  If they're remote, I can only see the instance they came from and I can go read the threads and assume whoever reported it was in the thread to try and derive the other side, but otherwise exceedingly difficult. (guess work - not only is it guess work, i shouldn't have to spend this much effort handling a single mod report - it's a scaling problem. It's like being a parent to two kids when both are wrong but one is more wrong than the other and both need to go to their room just to de-escalate. So that's why i do with deletes)

Not only are they one sided, but we don't know if the person making the mod report is acting offensively.  Are they causing arguments in order to get people mad to get them moderated?  Are they going and following people and looking to get them moderated after building trust and doing this with intention?   for example - If i see someone reported who has a private profile - my red flags go off on that report. You must intentionally make a connection - and if that person is remote - I can't do jack squat. But why would someone intentionally follow/approve just to moderate them when they should just unfollow and mute/block? Most of these reports are infighting, not rule breaking and its tiring... extremely tiring.  They turn into rule breaking when it results in public nastiness...  

So, with this one-sided-ness and bias towards privacy for the remote reporter, I tend to act swiftly with the delete button and If i can source the delete to the entire thread, I may do just that.   If the community is OK with swift justice, that's the way it should be.  "Calmer heads prevail" we say. (Or think) - but i don't think the community believes in this.  If i were to do this to someone whose goal is to debate hard topics in public, then i may delete their posts because of the one-sided-ness or private account policy or whatever it may be.  Should i be hanged and mercilessly shamed in the public eye because I acted swiftly with only the limited nonexistent information i do have?  I can suspend it if the reported person does things repeatedly (it's tracked) but i can't know if someone is a serial reporter - at all. Unless I remember it, or use the webhook to dump to a DB and try and track local users or people in threads - which gets expensive. 

That's the difficulty of moderating.  We don't have what is necessary to be nuanced, but quite frankly, being swift may be all we need.  "A social contract of swift moderation" if you will.

Some people will scream "hey, you're too big! if you were smaller you could moderate with nuance" - Nope...  The federation is the scale of moderation, not my local population.

Knowing that global moderation is the volume, and that one-sided-ness is the rule of law - for reasons - i think the best course of action is a global moderation understanding of bias towards simplicity.

Everyone should bias towards simplicity and any mod action should have reference to why it is simple.  

and we need an automoderator,  allow/deny lists, some basic filtering and stuff on admin side. It would catch a lot of easy stuff... I should just be able to deny *maga*, *qanon*, *antivax* in my usernames, trends, news but i can't easily do that.  I should be able to delay a post or do what twitter does pop a warning up, if it contains patterns of racially biased words - but we don't have any such thing... those posts get made, and without search - i must go read the DB to look for them and the last thing i want to do - is say i'm reading my DB so we end up waiting for user reports. As an admin, i pretend i'm working with financial data and I don't look. I don't need to look. I don't want to look.  SO whatever we can do to improve moderation so something is public to be seen or we can see both sides of the story would be a major improvement...  but this also empowers the bad admins and i think that's why we can't have anything good - the fear of providing tools that can be used offensively.

**Comment by supernovae** [comment]
Most mod activities aren't that eventful.

* Modding about random CW ethics.    I block 18+/porn... i ask people if they're being a thirst trap to CW that - but i can't CW Halloween, food, nature photos, Twitter reposts, and every random thing people want CW'd. It's a waste of my time - perhaps users should enable "CW all images in my profile" if they need this. 
* Modding about fights.  Don't fight. This is a user problem. If i see a fight, I'm going to delete it. I don't have any interest nor capability to referee a fight.   I can't tell the history, i can't tell what's going on, the only thing I can do is say "kids break it up" and clean up the mess.  This should be the expected behavior. Bias towards simplicity.
* Lots of porn... lots of it. 
* Random *MAGA* or nazi or fascist user... easy - delete.  I won't even read their appeal.  (Limited energy/time)
* Modding because users report the businesses who may be sharing their books/creations (please stop people, we need commerce)
* Modding because people hate the satire accounts and take offense to them
* modding because users try and actively attack celebrity accounts and hope they get internet points for causing someone to mod them.
* people who start their posts like "don't be a reply guy" and someone replies.. or "inb4" or some other poisoning the well...

A lot really are things where people should just unfollow and move on.  Mute and move on. Block and move on in most cases.  If someone is posting food you don't want to see - unfollow. It took you more time to report the post than it did to click the "mute" button during that same process...

I could go on and on if there was interest.   I don't need to debate CW ethics with other admins, i don't need to debate/referee fights with other admins, I don't need to debate why i don't want porn on my instance with other admins (and no its nothing against legal sex workers) and i certainly don't need MAGA users joining...  I feel the "mastodon covenant" covers some of this, but should be backed up with more tools/policy/procedure by those in the covenant.  That's where admins should have to work together as needed or they're not listed on joinmastodon

of course, there are problem users that we should go on the offense about and protect the community against... but most mod activities are people in the community who have a bad day, let something get to them, have a standard they want applied or they don't like something or don't want to see something.

I'm hopeful unlearing how we were on twitter helps..  but my grandma has a saying about hope :)

**Comment by ElanHasson** [comment]
Hi @thefuturebird, 

This is something we've discussed previously and have been discussing with several folks on various channels (👋🏼 @supernovae). Since you've started a discussion here, I figured this would be a great place to re-share here as well for a wider audience. Thanks for starting a thread!

I posted an initial draft of my thoughts after some discussions with various folks and have compiled an initial draft of my thoughts. I'm still working to incorporate the feedback but wanted to get more feedback.

You can see the initial draft and some of the raw feedback and clarifications that came out of the feedback. I'll update it all together after this round of feedback.

https://gist.github.com/ElanHasson/f2212425a459964a1210bda50806236d

Thanks!
